### ISSUE TITLE
Version 0.8.4 doesn't support Node13

### ISSUE THREAD

**Issue by Doc999tor** [TimelordUK/node-sqlserver-v8#125]
`src/bin` directory doesn't contain `v13` binaries

"msnodesqlv8": "^0.8.4",
"mssql": "^6.0.0"

```
C:\www\atzma.im\node_modules\msnodesqlv8\lib\bootstrap.js:64
        throw new Error('None of the binaries loaded successfully. Is your node version >= 10.15 ?')
              ^
Error: None of the binaries loaded successfully. Is your node version >= 10.15 ?
    at failIfNoBinaryExported (C:\www\atzma.im\node_modules\msnodesqlv8\lib\bootstrap.js:64:15)
    at Object.liveLoad (C:\www\atzma.im\node_modules\msnodesqlv8\lib\bootstrap.js:51:5)
    at Object.<anonymous> (C:\www\atzma.im\node_modules\msnodesqlv8\lib\bootstrap.js:80:12)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1103:10)
    at Module.load (internal/modules/cjs/loader.js:914:32)
    at Function.Module._load (internal/modules/cjs/loader.js:822:14)
    at Module.require (internal/modules/cjs/loader.js:956:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at C:\www\atzma.im\node_modules\msnodesqlv8\lib\connection.js:10:21```

**Comment by TimelordUK** [comment]
Hello.  I am looking at this unfortunately the current driver does not compile out the box with node version 13 so I am looking at what changes are required.  Hopefully will have a build in next 5 days.

**Comment by TimelordUK** [comment]
this is now available in 0.8.5
